apps/block_scout_web/lib/block_scout_web/views/api/v2/fee_payment_view.ex

```diff
@@ -0,0 +1,23 @@
+defmodule BlockScoutWeb.API.V2.FeePaymentView do
+  use BlockScoutWeb, :view
+
+  def render("message.json", assigns) do
+    ApiView.render("message.json", assigns)
+  end
```

Comment:
Is this needed? Same for `forward_transfer_view`

Reply:
Did a test, is not needed, removed from both views